stumpy/mstump.py

```diff
@@ -1228,7 +1228,9 @@ def mstump(
         err = f"T is {T_A.ndim}-dimensional and must be at least 1-dimensional"
         raise ValueError(f"{err}")
 
-    core.check_window_size(m, max_size=min(T_A.shape[1], T_B.shape[1]))
+    # mstump currently supports self-join. Therefore, the argument `n=T_A.shape[1]`
+    # should be passed to the function `core.check_window_size`.
```

Comment:
"must be passed"   

"should be" implies uncertainty and too passive

Reply:
Cool note! Understood 👍